libs/features/georouting/georouting.js

```diff
@@ -0,0 +1,162 @@
+// The flow chart for the georouting can be found here:
+// https://wiki.corp.adobe.com/display/WP4/GeoRouting
```

Comment:
Should be hosted in milo docs on sharepoint: `milo.adobe.com/docs/georouting`.

Reply:
done! - https://milo.adobe.com/docs/georouting/